data/data-pipeline/data_pipeline/etl/sources/child_opportunity_index/etl.py

```diff
@@ -0,0 +1,120 @@
+from pathlib import Path
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.score import field_names
+from data_pipeline.utils import get_module_logger, unzip_file_from_url
+
+logger = get_module_logger(__name__)
+
+
+class ChildOpportunityIndex(ExtractTransformLoad):
+    """ETL Child Opportunity Index data.
+
+    COI compiles a number of useful data sets. In the future, we could pull these
+    data sets in directly from their original creators.
+
+    Data dictionary available when you download zip from `self.COI_FILE_URL`.
+
+    Data source overview: https://data.diversitydatakids.org/dataset/coi20-child-opportunity-index-2-0-database.
+
+    Full technical documents: https://www.diversitydatakids.org/sites/default/files/2020-02/ddk_coi2.0_technical_documentation_20200212.pdf.
+
+    Github repo: https://github.com/diversitydatakids/COI/
+
+    """
+
+    def __init__(self):
+        self.COI_FILE_URL = (
+            "https://data.diversitydatakids.org/datastore/zip/f16fff12-b1e5-4f60-85d3-"
+            "3a0ededa30a0?format=csv"
+        )
+
+        self.OUTPUT_PATH: Path = (
+            self.DATA_PATH / "dataset" / "child_opportunity_index"
+        )
+
+        self.TRACT_INPUT_COLUMN_NAME = "geoid"
+        self.EXTREME_HEAT_INPUT_FIELD = "HE_HEAT"
```

Comment:
Researching heat stress’s effects on human health, it seems like it is indeed the absolute number of days over 90F that is the most scientifically consistent measure of heat impacts on health. Rather than measures like standard deviations above normal or anything like that. 

This impact is moderated by infrastructure like air conditioning or working indoors, which drastically reduces impact  of these heat days. That's why median house value is included since it correlates highly with air conditioning and there is no good AC data. The most recent national AC data is from the 1980 (!) census.

Reply:
Thanks @lucasmbrown-usds - this lines up well with [this study](https://ehjournal.biomedcentral.com/articles/10.1186/s12940-016-0102-7) that seems to be the top hit in PubMed